src/probnum/filtsmooth/gaussfiltsmooth/extendedkalman.py

```diff
@@ -20,35 +19,41 @@ def __init__(self, non_linear_sde, num_steps):
         self.num_steps = num_steps
 
     def transition_realization(self, real, start, stop, linearise_at=None, **kwargs):
+
         compute_jacobian_at = linearise_at.mean if linearise_at else real
-        jacobfun = functools.partial(
-            self.non_linear_sde.jacobian, state=compute_jacobian_at
-        )
+
+        def jacobfun(t, x=compute_jacobian_at):
+            # replaces functools (second variable may not be called x)
+            return self.non_linear_sde.jacobfun(t, x)
```

Comment:
now that jacobfun is a mere attribute, it is not ensured that the space variable is called state (it is entirely user specified). Thus, functools.partial is replaced with this wrapper.

Reply:
Sounds good :) I think you could remove the comments here and below, the functools reference is not that understandable anymore after this PR